app/models/manageiq/providers/ansible_tower/inventory/parser/automation_manager.rb

```diff
@@ -4,49 +4,99 @@ def parse
+      inventory_object[:name] = project.name
 
-      i.playbooks.each do |playbook_name|
+      project.playbooks.each do |playbook_name|
         # FIXME: its not really nice how I have to build a manager_ref / uuid here
-        p = target.playbooks.find_or_build("#{i.id}__#{playbook_name}")
-        # FIXME: how about just adding `o` - configuration_script_source here?
-        p[:configuration_script_source] = target.configuration_script_sources.lazy_find(i.id.to_s)
-        p[:name] = playbook_name
+        inventory_object_playbook = target.playbooks.find_or_build("#{project.id}__#{playbook_name}")
+        inventory_object_playbook[:configuration_script_source] = inventory_object
+        inventory_object_playbook[:name] = playbook_name
       end
     end
   end
+
+  def credentials
+    collector.credentials.each do |credential|
+      inventory_object = target.credentials.find_or_build(credential.id.to_s)
+      inventory_object[:name] = credential.name
+      inventory_object[:userid] = credential.username
+      # credential.description
+      # credential.host
+      # credential.password
+      # credential.security_token
+      # credential.project
+      # credential.domain
+      # credential.ssh_key_data
+      # credential.ssh_key_unlock
+      # credential.organization
+      # credential.become_method # '', 'sudo', 'su', 'pbrun', 'pfexec'
+      # credential.become_username
+      # credential.become_password
+      # credential.vault_password
+      # credential.subscription
+      # credential.tenant
+      # credential.secret
+      # credential.client
+      # credential.authorize
+      # credential.authorize_password
+      inventory_object[:type] = case credential.kind
+                                when 'net' then 'ManageIQ::Providers::AnsibleTower::AutomationManager::NetworkCredential'
+                                when 'ssh' then 'ManageIQ::Providers::AnsibleTower::AutomationManager::MachineCredential'
+                                when 'vmware' then 'ManageIQ::Providers::AnsibleTower::AutomationManager::VmwareCredential'
+                                # when 'scm' then 'ManageIQ::Providers::AnsibleTower::AutomationManager::???Credential'
+                                when 'aws' then 'ManageIQ::Providers::AnsibleTower::AutomationManager::AmazonCredential'
+                                when 'rax' then 'ManageIQ::Providers::AnsibleTower::AutomationManager::RackspaceCredential'
+                                when 'satellite6' then 'ManageIQ::Providers::AnsibleTower::AutomationManager::Satellite6Credential'
+                                # when 'cloudforms' then 'ManageIQ::Providers::AnsibleTower::AutomationManager::$$$Credential'
+                                when 'gce' then 'ManageIQ::Providers::AnsibleTower::AutomationManager::GoogleCredential'
+                                when 'azure' then 'ManageIQ::Providers::AnsibleTower::AutomationManager::AzureCredential'
```

Comment:
This class actually corresponds to `azure_rm` type as stated [here](https://github.com/jameswnl/manageiq/blob/440a6dc19bd8b97e55abc1aea67d72d30bbf7a0e/app/models/manageiq/providers/ansible_tower/automation_manager/azure_credential.rb#L1-L1).